pkg/cloudprovider/aws/apis/v1alpha1/provider_defaults.go

```diff
@@ -60,5 +60,5 @@ func (c *Constraints) defaultAMIFamily() {
 	if c.AMIFamily != nil {
 		return
 	}
-	c.AMIFamily = &AMIFamilyEKSOptimized
+	c.AMIFamily = &AMIFamilyAL2
```

Comment:
These AMIs aren't the plain AL2 images, but EKS specific AL2 images. Perhaps the naming should reflect that?

Reply:
The way I see it, we only want to support one AL2 family. The implementation happens to be EKSOptimized at the moment.